pkg/hostagent/hostagent.go

```diff
@@ -673,6 +667,7 @@ func (a *HostAgent) processGuestAgentEvents(ctx context.Context, client guestage
 	if err := client.Events(ctx, onEvent); err != nil {
 		return err
 	}
+	logrus.Info("closed")
```

Comment:
Should be `logrus.Debug`?

Reply:
Removed this. Added for debugging but not needed